recipes/qbs/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+from conans import ConanFile, tools
+from pathlib import Path
+
+class QbsConan(ConanFile):
+    name = "qbs"
+    description = "The Qbs build system"
+    topics = ("conan", "qbs", "build", "automation")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://qbs.io"
+    license = "LGPL-2.1-only", "LGPL-3.0-only", "Nokia-Qt-exception-1.1"
+    settings = {
+        "arch": ["x86_64"],
+        "os": [
+            "Linux",
+            "Macos",
+            "Windows"
+        ]
+    }
+    no_copy_source=True
+
+    def source(self):
+        tools.get(**self._data['sources'])
+
+    def build(self):
+        tools.download(**self._data['installer'], filename = "creator.7z")
+        self.run("cmake -E tar x creator.7z")
+
+    def package(self):
+        for p in self._data['package']:
+            self.copy(**p, symlinks=True)
+
+    def package_info(self):
+        binpath = Path(self.package_folder, self._data['bin_folder'])
+        self.env_info.PATH.append(str(binpath))
+
+    @property
+    def _data(self):
+        data = {
+            'Linux': {
+                'sources': self.conan_data['sources'][self.version][0],
+                'installer': self.conan_data['sources'][self.version][1],
+                'package': [
+                    {'dst': 'bin', 'pattern': 'bin/qbs*'},
+                    {'dst': 'bin', 'pattern': 'bin/qt.conf'},
+                    {'dst': 'bin', 'pattern': 'lib/qtcreator/libqbs*'},
+                    {'dst': 'bin', 'pattern': 'lib/qtcreator/plugins/qbs/*'},
+                    {'dst': 'bin', 'pattern': 'lib/Qt/lib/libic*'},
+                    {'dst': 'bin', 'pattern': 'lib/Qt/lib/libQt5Core*'},
+                    {'dst': 'bin', 'pattern': 'lib/Qt/lib/libQt5Gui*'},
+                    {'dst': 'bin', 'pattern': 'lib/Qt/lib/libQt5Network*'},
+                    {'dst': 'bin', 'pattern': 'lib/Qt/lib/libQt5Script*'},
+                    {'dst': 'bin', 'pattern': 'lib/Qt/lib/libQt5Xml*'},
+                    {'dst': 'bin', 'pattern': 'libexec/qtcreator/qbs*'},
+                    {'dst': 'bin', 'pattern': 'share/qtcreator/qbs/*'},
```

Comment:
share folder is not allowed in CCI. Remove it.

Reply:
This is already copied to the bin folder as suggested. `'dst': 'bin'`